### DISCUSSION TITLE
👩‍🌾 Bumpkin Competition #2

### DISCUSSION THREAD

**Discussion by BirbSunflower** [sunflower-land/sunflower-land#2552] [Art]
Attention all artists and fashion enthusiasts! 

Welcome to the second Bumpkin Competition! It's time to showcase your fashion skills and creativity to design wearables for our beloved Bumpkins! 🤩

There's no specific theme for this competition! Designers are encouraged to use their imagination and come up with original ideas. 

Before continuing, it is recommended to read the Bumpkin Guideline, which can be found here:

https://docs.sunflower-land.com/contributing/for-designers/bumpkin-guidelines

## Requirements ✏️

- Dimensions of file should be 100x100px.
- The layer (e.g. shirt) should be on it's own layer without any other parts. When placed on top of a Bumpkin it should perfectly align with its body.
- Art should be original and not plagiarised.
- If submitting multiple items, an aseprite file would be appreciated to speed up review. 🤗
- By submitting art, you acknowledge you are the original creator of the art and you give Thought Farm and the community the rights to use it freely in the game and marketing purposes indefinitely. Players may trade your item without the need to pay you royalties. The community may make changes and improve artwork that is submitted.
- You agree to the [terms and services](https://docs.sunflower-land.com/contributing/competitions/bumpkin-art-competition#terms-and-conditions) of the game.
- Bumpkins live inside of the game! Please include an NPC version of your design that can be used inside of Sunflower Land.

When submitting art, please include the following:

- Your farm ID.
- A minified version of your art piece.
- An example of the art used inside of the game or on the Bumpkin. Here's a nice submission example from a community artist: https://github.com/sunflower-land/sunflower-land/discussions/1748?sort=new#discussioncomment-5106261

## Rewards 🏆

The winner will receive 1000 SFL and a runner-up will receive 500 SFL, however this competition is not just about the prize money. It's an opportunity to showcase your incredible talent and become an essential part of the Sunflower Land experience. 

We want to emphasize that even if you don't win, your design may still have the chance to be selected in the future. Our team will carefully review all submissions and may reach out to designers whose work stands out for future collaboration opportunities. Sunflower Land is a community game and we love showcasing and including artwork from our players. Now is your chance to have your artwork immortalised in the #1 Community Web3 game.❤

## Deadline 📅

Designers have until May 29th to submit their entries. After that, our art team will carefully review and analyze each submission to select the winners.

The announcement of the winners will take place during the first week of June.

We can't wait to see what amazing designs you come up with. 
Good luck to all participants! 🌻

**Comment by N1491** [comment]
Farm id : 139032 discord name : jether | 139032 
 
![4](https://user-images.githubusercontent.com/121005517/236964782-f2e864fd-290e-48e2-b9f5-f06589d47c9b.png)
![1](https://user-images.githubusercontent.com/121005517/236964796-9757f7c8-28f0-46d1-9015-30f2cdcddc97.png)
![2](https://user-images.githubusercontent.com/121005517/236964803-40fac610-a48d-4c45-b633-eec4d90fce5d.png)

**Comment by future2back** [comment]
@BirbSunflower Sorry, I don't think this is the right place to discuss this, but will we insist on putting our efforts into Sunnyside design? When are we going to start a transition to an authentic identity of our own?
Thank you.

**Comment by yapiciemre** [comment]
`Farm ID : 7446 `
`Discord : Agnes#1353`


# **Information & NPC Preview**
Hat - Hair - Accessory - Top - Bottom - Shoes and Tool all made in a **_separate layer_**.
All of them are given together in the images below.
![Island300x300](https://github.com/sunflower-land/sunflower-land/assets/55670141/22df79fc-f272-4179-82e5-571fda08fa22)


## Aladdin Bumpkin
![Alaaddin300x300WB](https://user-images.githubusercontent.com/55670141/236452811-a3762a73-dd82-42ed-8c02-d1f8dc6d9982.png) ![Alaaddin100x100WB](https://user-images.githubusercontent.com/55670141/236452849-ad944ee3-466b-4550-a791-eb222c048257.png) ![Alaaddin300x300wBG](https://user-images.githubusercontent.com/55670141/236453000-7ae9de7b-685d-4bc8-9309-0424129da4b6.png) ![Alaaddin100x100wBG](https://user-images.githubusercontent.com/55670141/236453019-ab23bda6-868c-4b28-8ae1-0dd97813aca3.png)



## Jasmine Bumpkin
![Jasmine300x300WBG](https://user-images.githubusercontent.com/55670141/236453250-dac6cd6d-27e1-4f20-8d8d-167228e9f060.png) ![Jasmine100x100WBG](https://user-images.githubusercontent.com/55670141/236453263-22e6a5c3-8516-474a-8bc4-0628a444d709.png)

## Magical Night with Flying Carpet Background & Sand Tower Background
![Alaaddin300x300BG](https://user-images.githubusercontent.com/55670141/236453544-78647a7f-9150-423b-ba7d-5a3dad0c1a4e.png) &nbsp; ![Alaaddin100x100BG](https://user-images.githubusercontent.com/55670141/236453557-cfe6a7f7-8ca2-44cf-928a-680b3682a0de.png) &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; ![TowerBackgorund300x300](https://user-images.githubusercontent.com/55670141/236454133-9ab51a81-705b-4bcb-839c-ba4771647d9b.png) &nbsp; ![TowerBackgorund100x100](https://user-images.githubusercontent.com/55670141/236454140-28318097-b1d5-479f-9b3c-e32ecc830d07.png)

## Aladdin & Jasmine with Backgrounds
![Alaaddin300x300](https://user-images.githubusercontent.com/55670141/236455065-f6d2e540-b04b-4092-874b-7b58f1bbe39c.png) &nbsp; ![Alaaddin100x100](https://user-images.githubusercontent.com/55670141/236455142-114cb436-17ca-4752-9a18-dde99b646af9.png) &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; ![Jasmine300x300BG](https://user-images.githubusercontent.com/55670141/236455516-f7d36be9-2a59-44c4-a15c-2dc58edf9dee.png) &nbsp; ![Jasmine100x100BG](https://user-images.githubusercontent.com/55670141/236455549-6bfe0dfd-1eae-47ea-b013-6b90c4701a1b.png)


![AlaaddinSBG300x300](https://user-images.githubusercontent.com/55670141/236457302-a36cd090-a675-4d1d-8708-d53c62de9235.png) &nbsp; ![AlaaddinSBG100x100](https://user-images.githubusercontent.com/55670141/236457323-f46d6c4b-5833-498e-9af5-2c9f9d145a02.png) &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; ![JasmineSBG300x300](https://user-images.githubusercontent.com/55670141/236457441-cf274cef-fa24-47ec-9af0-d6bada5480bf.png) &nbsp; ![JasmineSBG100x100](https://user-images.githubusercontent.com/55670141/236457451-fbe009a0-9552-44fd-992a-29cf7a207f07.png)

**Comment by sfwhat** [comment]
### Genie from the lamp!


<img width="591" alt="image" src="https://github.com/sunflower-land/sunflower-land/assets/95937808/d2d52588-7a6f-4f98-866a-b0dc03ac935a">

![genie2](https://github.com/sunflower-land/sunflower-land/assets/95937808/0b649165-852d-48cc-ab3e-7cb9cd92cc5b)


Sfwhat#8844

**Comment by peperol** [comment]
Baby Eggplant
Discord Peperol#2715
farm ID 145849
![melanzana](https://github.com/sunflower-land/sunflower-land/assets/114354077/2b801c4b-24ea-4db8-84b6-99c072ca64f1)

shirt
![shirt mel](https://github.com/sunflower-land/sunflower-land/assets/114354077/b0c55c09-8637-4038-83f4-fbd4aaa58512)
skirt 
![skirt rabb](https://github.com/sunflower-land/sunflower-land/assets/114354077/cbb25af2-2b49-4bb3-af1e-6f4754140407)

eggplant hair
![hairmela](https://github.com/sunflower-land/sunflower-land/assets/114354077/04e3c3d0-8652-4b34-af90-7ff74a804323)

![melanzino3](https://github.com/sunflower-land/sunflower-land/assets/114354077/20ce82e1-cf0c-44e8-93be-6f56ebc208a7)
![melanzana in land](https://github.com/sunflower-land/sunflower-land/assets/114354077/4181b4bb-529e-4d56-90e1-900d51dbc52a)

Boy Eggplan Fan
![Immagine 2023-05-16 120342](https://github.com/sunflower-land/sunflower-land/assets/114354077/dbe6a6e9-ef07-4967-a26c-d65d38c809ad)
hoodie
![hoodie](https://github.com/sunflower-land/sunflower-land/assets/114354077/8ceab6e9-4db4-4aca-a194-696e0f9a1c08)
pants
![pants4](https://github.com/sunflower-land/sunflower-land/assets/114354077/8724a252-ee89-4379-b9d1-f805c2052727)
shoes
![scarpe](https://github.com/sunflower-land/sunflower-land/assets/114354077/92f993ca-05e1-49db-9019-656c131ee72f)
trumpet
![tromba](https://github.com/sunflower-land/sunflower-land/assets/114354077/dc5c53bb-5724-4676-9533-84eaab40f814)


![melanzino4](https://github.com/sunflower-land/sunflower-land/assets/114354077/5c5bb9a8-94bc-4afb-bf7a-ce1891f0c77a)
![Landboymel](https://github.com/sunflower-land/sunflower-land/assets/114354077/1ffbb8a6-cb5d-4c60-8171-a5d4bc953b64)

sunflower boy fan
![sunf boyland](https://github.com/sunflower-land/sunflower-land/assets/114354077/5ef8c34e-f928-4543-a922-4e12cf55b622)

hoodie
![hoodiesunflo](https://github.com/sunflower-land/sunflower-land/assets/114354077/5fb81ca0-779c-45d2-95f2-38fc17306c49)

sunflower bear 
![sunflowerbear](https://github.com/sunflower-land/sunflower-land/assets/114354077/4553a4c2-d879-4cd4-96bb-cbd64e24ba11)

![sunflower boy](https://github.com/sunflower-land/sunflower-land/assets/114354077/cb44a273-30fc-4036-8194-0a11e665223c)
![Land6e boy sunflower](https://github.com/sunflower-land/sunflower-land/assets/114354077/e4599a60-673b-4ffc-bb40-25aa956d1446)

the Queen
![queenscren](https://github.com/sunflower-land/sunflower-land/assets/114354077/1d0a3c36-5e69-4b2c-bab3-e4361734b449)

Queen coat
![queencoat](https://github.com/sunflower-land/sunflower-land/assets/114354077/d3144772-03b6-4f12-abe2-fdada8d13a7a)
Queen dress
![dress](https://github.com/sunflower-land/sunflower-land/assets/114354077/474a7088-509a-495a-b940-13e2ce1a3433)

hair
![hair](https://github.com/sunflower-land/sunflower-land/assets/114354077/d885791d-3abd-4a8e-bc06-74ec7f35824d)
Crown
![diadema](https://github.com/sunflower-land/sunflower-land/assets/114354077/41629aa3-b08d-4063-8dc2-1f9d4a8b48a7)
shoes
![scarpequeen](https://github.com/sunflower-land/sunflower-land/assets/114354077/4cd25f2a-336a-4e34-add6-d264fb8887e6)


![queen](https://github.com/sunflower-land/sunflower-land/assets/114354077/ab995773-8e6a-4df7-9b04-e4a05dbd6e84)

![queen in land](https://github.com/sunflower-land/sunflower-land/assets/114354077/20af4118-a389-49ff-83ed-cadda93efce3)

pirate woman
![pirate woman](https://github.com/sunflower-land/sunflower-land/assets/114354077/9c5c99b1-b8e3-4e76-a5a2-084daed817d8)


top
![pirate top](https://github.com/sunflower-land/sunflower-land/assets/114354077/e709efb7-4f28-48ba-92c8-62172f5aed53)
skirt
![pirate skirt](https://github.com/sunflower-land/sunflower-land/assets/114354077/db57348e-edb7-488c-9c0a-b2b6963105f5)
curly hair
![curly hair](https://github.com/sunflower-land/sunflower-land/assets/114354077/08bafeee-c991-4ead-b110-e3f20fd56e21)
bandana
![pirate bandana](https://github.com/sunflower-land/sunflower-land/assets/114354077/bfeb015f-2a64-4d65-8080-58a12be79315)
Gold nugget necklace
![pepita neklace](https://github.com/sunflower-land/sunflower-land/assets/114354077/0e99892e-01f3-47a0-859a-60d71254c1d5)
stiletto
![pirate woman shoes](https://github.com/sunflower-land/sunflower-land/assets/114354077/c330abd6-e0fd-4581-92a0-73dd7be5be98)
pirate gun
![pirate gun](https://github.com/sunflower-land/sunflower-land/assets/114354077/e6cd86e1-341f-4da5-b2f2-4160da21b14c)

![pirate](https://github.com/sunflower-land/sunflower-land/assets/114354077/b3276d07-dda7-4b95-aa07-a70a336178d0)
![pirate woman in land](https://github.com/sunflower-land/sunflower-land/assets/114354077/568c3bc3-b683-4419-8d58-84b85fa6eb53)

**Comment by Vergelsxtn** [comment]
### The Mysterious Explorer ✨
**Farm ID: 63** 🌻

**BUMPKIN:**
![Explorer-Bumpkin-Prev](https://github.com/sunflower-land/sunflower-land/assets/99605039/960c5233-f0b2-4467-89fe-383ad2121c52)

| **Shirt** | **Pants** | **Boots** | **Back Pack** | **Magic Wand** |
|--------|--------|--------|--------|--------|
| ![Explorer-shirt](https://github.com/sunflower-land/sunflower-land/assets/99605039/51832575-bfc2-46b2-a579-d4b4273be005) | ![Explorer-pants](https://github.com/sunflower-land/sunflower-land/assets/99605039/b5dc5aa6-c3ba-433d-a1a1-025765e67c3f) | ![Explorer-boots](https://github.com/sunflower-land/sunflower-land/assets/99605039/6da2b4da-bd35-418a-af2e-c9624c320908) | ![Explorer-backpack](https://github.com/sunflower-land/sunflower-land/assets/99605039/c0479d28-5388-475f-8def-77510c9f1950) | ![Explorer-MagicWand](https://github.com/sunflower-land/sunflower-land/assets/99605039/d7089e2c-dda2-4db9-b595-266276d2693f) |

**IN-GAME:**
![Explorer-npc-prev](https://github.com/sunflower-land/sunflower-land/assets/99605039/6c1b4975-887d-4984-862d-ea27ec5422fc)
![explorernpc](https://github.com/sunflower-land/sunflower-land/assets/99605039/0d492854-2f0b-41cc-994a-985a828df239)

| Sprite Sheet |
|--------|
| **Back Pack:** ![Explorer-sprite-backpack](https://github.com/sunflower-land/sunflower-land/assets/99605039/6ed39bea-6e86-4302-a838-e68622124d74) |
| **Shirt:** ![Explorer-sprite-shirt](https://github.com/sunflower-land/sunflower-land/assets/99605039/8ec6a53b-7f9a-4e9a-9745-679cf925877b) |
| **Pants:** ![Explorer-sprite-pants](https://github.com/sunflower-land/sunflower-land/assets/99605039/f62830c8-da6a-42d7-b227-6dae30e41688) |

**Comment by Thebyber** [comment]
**Monkey di goblin**

**Farm id:** 16230

Bumpkin:
![image](https://github.com/sunflower-land/sunflower-land/assets/124185010/4405f52e-8690-418b-9e9f-9c46370085e2)


**In game:** 
![monkey d goblin](https://github.com/sunflower-land/sunflower-land/assets/124185010/06e8f91e-27ef-4362-ad92-c0456e514e70)

**Comment by esp123** [comment]
Hello... again XD!
Random submissions hehe....
![hoothoot](https://github.com/sunflower-land/sunflower-land/assets/74337946/72509c27-6c93-464c-bb6e-1aed8d406814)

Nest BG
![nestbgx7](https://github.com/sunflower-land/sunflower-land/assets/74337946/eddef477-20a5-4d5c-97ce-9fb74a5769cc)
![nestx1](https://github.com/sunflower-land/sunflower-land/assets/74337946/22e0a9a6-df4a-48a9-9c75-921c7cfff9d6)

Egg Suit
![eggsuitx7](https://github.com/sunflower-land/sunflower-land/assets/74337946/223d96b3-8dff-4365-b268-7ff9cc834173)
![eggx1](https://github.com/sunflower-land/sunflower-land/assets/74337946/2b179cc3-eead-4511-823f-994bd27330a6)
![npc-egg](https://github.com/sunflower-land/sunflower-land/assets/74337946/88d4b1ba-bbac-4867-9338-5ac8b5679464)

Worm
![wormx7](https://github.com/sunflower-land/sunflower-land/assets/74337946/e56d7a31-a7bb-4b2c-ad5f-5fec50d2f060)
![wormx1](https://github.com/sunflower-land/sunflower-land/assets/74337946/9429915c-d31a-444c-ba0e-1c9a9a8b03cf)

Hoot Hat
![hootx7](https://github.com/sunflower-land/sunflower-land/assets/74337946/5de09110-49df-47d4-a3f0-068108d31271)
![hootx1](https://github.com/sunflower-land/sunflower-land/assets/74337946/59f77cce-a568-4f35-9793-30786bdc3c08)
![npc-hoot](https://github.com/sunflower-land/sunflower-land/assets/74337946/83a3c577-b419-4de7-91ba-2dabd2ba6f88)

Centaur 1
![centaur](https://github.com/sunflower-land/sunflower-land/assets/74337946/30188b51-d1e8-4251-a3a6-5643438a07ae)
![centaurlegsx1](https://github.com/sunflower-land/sunflower-land/assets/74337946/30dfe139-c86d-4b30-9476-0460aaff654f)
Centaur 2
![centaur2](https://github.com/sunflower-land/sunflower-land/assets/74337946/3ecfd9e1-e92c-4299-b67a-de21ee9d0fff)
![centaur2legsx1](https://github.com/sunflower-land/sunflower-land/assets/74337946/7767756a-899a-47b9-9933-87e93096349d)
![npc-cent](https://github.com/sunflower-land/sunflower-land/assets/74337946/488be07e-f9b6-412e-ab4e-90bc26a5b914)

Satyr (Goat Legs)
![satyr](https://github.com/sunflower-land/sunflower-land/assets/74337946/d35e603d-4a53-43e9-a9f9-4623d5290f0e)
![satyrlegsx1](https://github.com/sunflower-land/sunflower-land/assets/74337946/8d352474-31ba-4d31-87dc-8a307452a767)
![npc-satyr](https://github.com/sunflower-land/sunflower-land/assets/74337946/e945c99f-64f5-43f3-8bb3-71cf75efc9a8)

Crab Legs
![crab](https://github.com/sunflower-land/sunflower-land/assets/74337946/05e833ca-e5b9-405e-b013-ea0ae948020d)
![crablegsx1](https://github.com/sunflower-land/sunflower-land/assets/74337946/70067227-2483-4e83-994a-189e7115da21)
![npc-crab](https://github.com/sunflower-land/sunflower-land/assets/74337946/5eabf4ce-d494-4054-b160-694eba5e285e)

Bonus: 
Shoulder Spaulder
![shoulder](https://github.com/sunflower-land/sunflower-land/assets/74337946/6ddb3936-9f5c-485d-a109-6eab5d612683)
Starry Night BG
![starryx1](https://github.com/sunflower-land/sunflower-land/assets/74337946/139f95da-a943-4c48-acd1-ad597a8c024d)

Whale Hat
![whaley](https://github.com/sunflower-land/sunflower-land/assets/74337946/9fba8670-8d14-46ae-84b7-00a38b2b4458)
![whaleey](https://github.com/sunflower-land/sunflower-land/assets/74337946/f2958ea3-00ae-4774-93d4-ddfebebca825)
![npc-whale](https://github.com/sunflower-land/sunflower-land/assets/74337946/f23cdf7f-97f1-4680-b7c7-7be7d4252d01)


Discord: deeefault#0033
Farm id: 80399

Feel free to leave some suggestions or feedbacks ❤️
